documentation/blog/2025-03-26-mcp-security/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to Determine If An MCP Server Is Safe"
+description: Connect Goose directly to your code editor with this Visual Studio Code MCP.
+authors: 
+    - ebony
+---
+
+![blog cover](mcpsafety.png)
+
+# How I Vet MCP Servers Before Plugging Them In
+
+MCP servers are everywhere right now, last time I checked there was over **2,600 and counting**. Every day, a new one pops up, letting AI agents like Goose access files, query your Google Drive, search the web, and unlock all kinds of wild integrations.
```

Comment:
I think this is a comma splice
```suggestion
MCP servers are everywhere right now. Last time I checked, there were over **2,600 and counting**. Every day, a new one pops up, letting AI agents like Goose access files, query your Google Drive, search the web, and unlock all kinds of wild integrations.
```

Another suggestion could be to say 
MCP servers are everywhere right now. As I'm writing this, there are over 2,600 or something like that.